SSA/Experimental/Bits/Fast/BitStream.lean

```diff
@@ -437,7 +473,6 @@ theorem add_congr (e1 : a ≈ʷ b) (e2 : c ≈ʷ d) : (a + c) ≈ʷ (b + d) := b
     · simp only [addAux, e1 _ h, e2 _ h, ih (by omega)]
   simp [HAdd.hAdd, Add.add, BitStream.add, add_congr_lemma, addAux]
 
-
 theorem not_congr (e1 : a ≈ʷ b) : (~~~a) ≈ʷ ~~~b := by
```

Comment:
This is unrelated. Drop this change?

Reply:
This dropped line is still there. I suggest to do this cleanup in a separate PR (no review required).